### PR TITLE
[release-4.14] OCPBUGS-44095: Backport SDN live migration

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

**Comment by pliurh** [comment]
/hold
depends on https://github.com/openshift/api/pull/1994

File: .vscode/settings.json
Comment (JoelSpeed) [inline .vscode/settings.json:1]: I don't think this is intentionally checked in

File: go.mod [hunk 1]
```diff
@@ -115,3 +115,5 @@ require (
 	sigs.k8s.io/kube-storage-version-migrator v0.0.6-0.20230721195810-5c8923c5ff96 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 )
+
+replace github.com/openshift/api => /root/projects/api
```
Comment (JoelSpeed) [inline go.mod:119]: Can we get this to a fixed commit in some repository rather than to a local folder

Comment (pliurh) [reply go.mod:119]: This is for creating private build for QE testing. I'll bump the api version once the API PR is merged.

**Pull request merged**